cpp/src/glm/ridge.cuh

```diff
@@ -187,7 +188,19 @@ void ridgeFit(const raft::handle_t& handle,
                    norm2_input.data(),
                    fit_intercept,
                    normalize,
-                   stream);
+                   sample_weight);
+  }
+  if (sample_weight != nullptr) {
```

Comment:
It would be nice to see the gtests handing a set of sample weights, even if just a uniform distribution, just to make sure the path through the C+ code is being covered.